frontend/src/components/pages/RegistrantExperience/WaiverPage/index.tsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import {
+  Box,
+  Text,
+  Heading,
+  VStack,
+  StackDivider,
+  Stack,
+  Checkbox,
+  ListItem,
+  OrderedList,
+} from "@chakra-ui/react";
+import { ReactJSXElement } from "@emotion/react/types/jsx-namespace";
+import {
+  OptionalClauseResponse,
+  RequiredClauseResponse,
+  WaiverActions,
+  WaiverInterface,
+} from "./waiverTypes";
+
+interface WaiverPageProps {
+  waiverInterface: WaiverInterface;
+  waiverDispatch: any;
+}
+
+const WaiverPage = ({
```

Comment:
> what's the motivation behind the rename? think our original naming was so that the "page" (`RegistrantExperiencePage`) referred to the parent component with the stepper and footer

I was previously importing the `Waiver` type from `AdminTypes.ts` which had name conflicts with the `<Waiver>` component. I noticed that I'm no longer importing the `Waiver` type so I'll revert the `<WaiverPage>` component back to `<Waiver>`.

Reply:
> though maybe we should rename that something less verbose like `RegistrationPage`

Yep, will do.